core/chains/orm.go

```diff
@@ -10,13 +10,13 @@ import (
 	"github.com/smartcontractkit/chainlink/core/services/pg"
 )
 
-type ChainsORM[I ID, CFG Config, C ChainConfig[I, CFG]] interface {
+type ChainConfigs[I ID, CFG Config, C ChainConfig[I, CFG]] interface {
```

Comment:
I'll look through the names again. I was trying to avoid collisions. Things should get simpler once we can fully remove the ORMs after updating the DB schema.

Reply:
Hmm do you have any specific suggestions? We're kind of stuck with an extra layer until the DB is removed.